pkg/controller/common/certificates/ca_test.go

```diff
@@ -46,7 +47,9 @@ var (
 )
 
 func init() {
-	logf.SetLogger(logf.ZapLogger(false))
+	logf.SetLogger(crzap.New(func(o *crzap.Options) {
```

Comment:
Looks like we missed this unnecessary logger initialisation during the earlier refactoring. Do you mind getting rid of it?

Reply:
done!